### ISSUE TITLE
RAR Extraction with Chinese/Japanese characters in password fails

### ISSUE THREAD

**Issue by jtiiii** [aonez/Keka#335]
It's spent 1hours on extracting.
![837fb5889b3fb9fd114cff69127f1f81](https://user-images.githubusercontent.com/25175725/49140018-2b948900-f32e-11e8-9af8-c2e0b967ef00.jpg)
That rar file's size has over 10GB. But I think not spent so much time.

Finally , I extract it by WINRAR 5.0。
The file owner said it use RAR5 format to compression.
I think is problem.

**Comment by aonez** [comment]
Keka supports RAR v5, so the problem must be other. Looking at the screenshot, I assume the progress bar maybe did not gave any actual progress? If it's so, there must be another problem.

Can you enable the [console log](https://github.com/aonez/Keka/wiki/Console-verbose), try again and let me know what it says in the Console.app?

```sh
defaults write com.aone.keka DevLog -bool true
defaults write com.aone.keka DevLogReader -bool true
```

**Comment by aonez** [comment]
By the way no need to wait, just launch the process with the console log enabled and there must be some output in the first seconds.

**Comment by aonez** [comment]
@jtiiii just did a test with a 10+GB parted v5 RAR file with no problem. So I'm sure there was another problem, maybe a file access one, that Keka did not handled properly and the process hang there. So will be interesting to see the console output if you can replicate this one.

**Comment by aonez** [comment]
@jtiiii just found that this version has two bugs, one that prevents RAR progress to be shown in the progress bar, and another that makes a bad password RAR extraction not being detected. So ignore all the previous messages. I suppose you mistyped the password and the task never needed. Sorry about that one.

**Comment by aonez** [comment]
@jtiiii can you check this version works with that file: https://t.keka.io/Keka.r3153-rar.7z

**Comment by aonez** [comment]
Thanks @jtiiii. It seems that also compression with this characters fail.

**Comment by aonez** [comment]
> cannot support Chinese passwords

Will be fixed in 1.1.8.

**Comment by aonez** [comment]
@jtiiii can you confirm this new build works: https://t.keka.io/Keka.r3157-rar.7z

**Comment by aonez** [comment]
Should be fixed in 1.1.8.
